avm/res/cognitive-services/account/main.bicep

```diff
@@ -341,7 +341,7 @@ type diagnosticSettingType = {
   }[]?
 
   @description('Optional. A string indicating whether the export to Log Analytics should use the default destination type, i.e. AzureDiagnostics, or use a destination type.')
-  logAnalyticsDestinationType: ('Dedicated' | 'AzureDiagnostics' | null)?
+  logAnalyticsDestinationType: ('Dedicated' | 'AzureDiagnostics')?
```

Comment:
Todo as discussed:
- add version bump
- update reference to cross referenced modules
- regenerate readmes with latest utility

Wait for #597

Reply:
Should be done @eriqua